config/routes.rb

```diff
@@ -6,12 +6,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-  resources :categories do
+  resources :categories, only: %i[index] do
     resources :products, only: %i[index], controller: 'categories/products'
   end
-  resources :users
-  resources :banners
-  resources :assets
-  resources :products
-  resources :addresses
+  resources :banners, only: %i[index]
+  resources :products, only: %i[index show]
+  resources :addresses, only: %i[index show create update destroy]
```

Comment:
Este se puede quedar así, es lo mismo.

```suggestion
  resources :addresses
```